docs/docs/tutorials/spring-boot-integration.md

```diff
@@ -187,6 +187,35 @@ In this case, you must explicitly specify **all** components.
 
 More details can be found [here](https://github.com/langchain4j/langchain4j-spring/blob/main/langchain4j-spring-boot-starter/src/main/java/dev/langchain4j/service/spring/AiService.java).
 
+### Listening for AI Service Registration Events
+
+After you have completed the development of the AI Service in a declarative manner, you can also listen for the
+`AiServiceRegisteredEvent` by implementing the `ApplicationListener<AiServiceRegisteredEvent>` interface.
+
+This event is triggered when AI services are registered in the Spring context, 
+allowing you to obtain information about all registered AI services and their tools at runtime. 
+This provides the application with dynamic configuration, debugging, and extension capabilities,
+making it particularly suitable for scenarios that require specific actions based on service registration status.
+Here is an example:
+```java
+@SpringBootApplication
+class AiServiceWithToolsApplication implements ApplicationListener<AiServiceRegisteredEvent> {

```

Comment:
nit: `ApplicationListener` should probably not be a `@SpringBootApplication`, perharps make it a regular component?

Reply:
done